src/_macosx.m

```diff
@@ -1122,7 +1128,7 @@ - (void)close
 {
     [super close];
     --FigureWindowCount;
-    if (!FigureWindowCount) [NSApp stop: self];
+    if (!FigureWindowCount) stop(NULL);
```

Comment:
This leaks the `None` reference returned from `stop` (at least on versions before it became immortal).

Reply:
👍 Added a decref to the return object now. I could change this back to the original `[NSApp stop: self]` and add the `PyOS_InputHook = NULL` call here too instead if that is better? I just figured it might be nice to try and go through one stop function everywhere we need it, although it is only in two places currently.